docs/usage/getting-started/running.md

```diff
@@ -1,68 +1,95 @@
 # Running Renovate
 
-As a Renovate end user, there are two main categories of use:
+As end user, you can choose from these ways to run Renovate:
 
-- You self-host Renovate, e.g. by running the pre-built Docker image, or
+- You use the Mend Renovate app on GitHub, or
+- You self-host Renovate, for example by running the pre-built Docker image, or
 - Someone else is hosting Renovate, and you install/configure it for the repositories you choose
 
-If someone else is hosting Renovate for you, or you are using the Mend Renovate App on GitHub, then you can skip ahead to the [installing & onboarding](./installing-onboarding.md) page.
+If you're using the Mend Renovate App on GitHub, or if someone else is hosting Renovate for you, skip ahead to the [installing & onboarding](./installing-onboarding.md) page.
 
 ## Self-Hosting Renovate
 
-Self-hosting Renovate means that you are the "administrator" of the bot, which entails:
+When self-hosting Renovate you're the "administrator" of the bot, this means you:
 
-- You provide infrastructure for it to run on,
-- You provision its global config,
-- You ensure it's running regularly,
-- You ensure Renovate bot itself is updated
+- provide the infrastructure that Renovate runs on,
+- provision Renovate's global config,
+- make sure Renovate bot runs regularly,
+- make sure Renovate bot itself is updated
 
-Read our section on [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
+If you're self-hosting Renovate on Windows, read [Self-hosting on Windows](./installing-onboarding.md#self-hosting-on-windows) to prevent line endings from confusing Renovate bot.
 
 ### Available distributions
 
 #### npm package (CLI)
 
 Renovate's Open Source CLI is built and distributed as the npm package `renovate`.
-You can run this directly in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
-When you install Renovate from npm it naturally does not come bundled with any third-party tools or languages such as Ruby, Python, Composer, Bundler, Poetry, etc.
+You can run this package in any Node.js environment - even via `npx` - and it will process all the repositories it is configured with, before exiting.
+
+Renovate's npm package does not bundle any third-party tools or languages like Ruby, Python, Composer, Bundler, Poetry, etc.
 If you need Renovate to support any non-npm lock files like Bundler then you'll need to make sure all required third-party tools are pre-installed in the same environment alongside Renovate before you run it.
```

Comment:
```suggestion
By default, Renovate does not support non-npm lock files.
You must install all required third-party tools in the same environment that Renovate's going to run in, _before_ running Renovate.
For example: to use Bundler's `Gemfile.lock` you would first install the Bundler package manager into Renovate's environment.
```

Whew, that was a tricky rewrite. Let's see what you think of this. 😄 

> in the same environment that Renovate's going to run in, _before_ running Renovate.

This part still feels a bit clunky though, open for suggestions!

Reply:
Let's just say "lock files" and keep it generic - ignoring that npm is usually installed already

Reply:
```suggestion
By default, Renovate does not support non-npm lock files.
You must install all required third-party tools in the same environment that Renovate's going to run in, _before_ running Renovate with non-npm lockfiles.
```

Do you mean like this?